cmd/update.go

```diff
@@ -58,6 +58,11 @@ func updateBuildAgent(cmd *cobra.Command, args []string) error {
 
 // Still depends on a function in cmd/build.go
 func findLatestPicardSha() (string, error) {
+
+	if err := ensureDockerIsRunning(); err != nil {
```

Comment:
Maybe this could be called from a function that creates the `Command` so that it's a bit more obvious when it needs to be called?

```go
func DockerCommand(args ...string) (*exec.Cmd, error) {
    if err := ensureDockerIsRunning(); err != nil {
        return nil, err
    }
    return exec.Command("docker", args...), nil
}
```